src/qt/rpcconsole.h

```diff
@@ -108,7 +108,7 @@ public Q_SLOTS:
     /** Set network state shown in the UI */
     void setNetworkActive(bool networkActive);
     /** Set number of blocks and last block date shown in the UI */
-    void setNumBlocks(int count, const QDateTime& blockDate, double nVerificationProgress, bool headers);
+    void setNumBlocks(int count, const QDateTime& blockDate, double nVerificationProgress, bool headers, bool initial_sync);
```

Comment:
In commit "relax GUI locks: avoid unnecesarry calls to ::ChainstateActive().IsInitialBlockDownload()" (feecf490c418399e9771a5b48e078a4c2c5b6b55)

Could revert all these changes to rpcconsole.h and rpcconsole.cpp. Qt is fine with dropping ununused signal arguments https://doc.qt.io/qt-5/signalsandslots.html "a slot may have a shorter signature than the signal it receives"